packages/fxa-auth-server/lib/senders/emails/templates/welcome.ts

```diff
@@ -0,0 +1,16 @@
+export const render = () => {
+  return `
+    <mjml>
+      <mj-body>
+      <mj-include path="./lib/senders/emails/css/style.css" type="css" css-inline="inline" />
```

Comment:
It was mentioned in the [issue](https://github.com/mozilla/fxa/issues/9247#issuecomment-868866887) if we can make use of a `.css` file instead of `typescript` file for styling so this change addresses that concern